### ISSUE TITLE
PPOCR Cannot recognise capital letters and punctuations

### ISSUE THREAD

**Issue by koayst** [PaddlePaddle/PaddleOCR#2346]
Hi,

Built a rec model using configs/rec/rec_icdar15_train.yml.  I made some modifications to it.  For example:

pretrained model: I used the one downloaded from (https://paddleocr.bj.bcebos.com/dygraph_v2.0/en/rec_mv3_none_bilstm_ctc_v2.0_train.tar)
- pretrained_model: pretrain_models/rec_mv3_none_bilstm_ctc_v2.0_train/best_accuracy

-character_dict_path: ppocr/utils/ic15_dict.txt adds to the end of the file capital letters "A" to "Z" and some punctuation characters like comma, full-stop, semi-colon, question mark, colon for example.

Trained up to 200 epoch and the best metric acc is 0.75507.

When I tested the model, there is no capital letter and punctuation mark in the prediction even thought the training images have Big/Small characters and punctuations marks.  For example date like 01/01/2021.

image is "JOINT" ==> prediction is "joint"
image is "REMARK" ==> prediction is "remark"
image is ":" ==> prediction is "a"
image is "03:45" ==> prediction is "0345"

Q1: How can I fix this issue ?
Q2: What is the best pre-trained model and YML file combination to properly train a rec model so as to recognise a page of English text (including punctuations) only ?

**Comment by alenma04** [comment]
Hi @koayst you can try universal Chinese model with language as "ch" itself to detect English text along with special characters.

**Comment by koayst** [comment]
OK.  Thanks.  I will go try.

**Comment by Ankur-singh** [comment]
@koayst is the universal Chinese model working better than the English model?

I have the same issue, I need a case sensitive model, that should also be able to detect English numbers
